content/docs/1.1.1/advanced-resources/rancher-cluster-restore.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Restore cluster with a Rancher snapshot
+weight: 4
+---
+
+This doc describes what users need to do after restore the cluster with a Rancher snapshot.
+
+## Assumptions:
+- Most of the data still exist in the cluster before the restore and can be directly reused then.
+- There is a backupstore holding all volume data.
+- The setting [`Disable Revision Counter`](../../references/settings/#disable-revision-counter) is false. (It's false by default.) Otherwise, users need to manually check if the data among volume replicas are consistent, or directly restore volumes from backup.
+
```

Comment:
Is the longhorn system restore have some caveat?

Like it only allows disaster recovering scenario, which means restoring to the same cluster (the same node size and volume size)?
Or is the longhorn system restore support cluster migration scenario?

Reply:
Yeah. For this issue, we focus on the disaster recovery scenario only. And this issue is like an instruction that helps users bring as many volumes back as possible, rather than a new feature. Hence there won't be any warning after users restore the whole Longhorn system or the cluster.